src/components/ThreeDEditor.js

```diff
@@ -46,6 +46,8 @@ export class ThreeDEditor extends React.Component {
                 atomRadiiScale: 0.2,
                 atomRepetitions: 1,
             },
+            originalMaterial: this.props.material,
+            material: this.props.material.clone(),
```

Comment:
Let's add a comment explaining the logic for using originalMaterial and material

Reply:
Done.